src/components/pages/Authentication/SignUp/index.tsx

```diff
@@ -1,20 +1,21 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
 import * as React from 'react';
 import { connect } from 'react-redux';
 
 import SignUpForm from './SignUpForm';
 import withAuthenticationContainer, { AuthenticationProps } from 'components/containers/AuthenticationContainer';
-import __spacing from 'settings/__spacing';
 import { signUp } from 'store/actions/auth';
 
-function SignUp(props: AuthenticationProps) {
- 
-   return (<SignUpForm {...props} />);
+export interface SignUpProps extends AuthenticationProps {
```

Comment:
I think this actually makes a lot of sense but would `LoginProps` not have the same value? Is it possible to `isLoading` to `AuthenticationProps`and then perform this inheritance only for `NewPasswordProps`( which would now have only the `changePassword` key that was added to it)

Reply:
That's right. But I'm also looking at the context in which the `isLoading` prop exists. It is available on those components but not on the Authentication component as well. Adding it there I feel will just add one more prop which will never exist in the Authentication context for the sake of `prop drilling`.

Reply:
> Adding it there I feel will just add one more prop which will never exist in the Authentication context 

If by `Authentication context ` you mean the prop in `pages/Authentication/index.tsx`, that is actually totally different. I dropped a comment to rename of the interface for that component's prop.

If not, then the `AuthenticationProps` that we are talking about here would be used in 4 places:
- SignUp
- Login
- ForgotPassword
- ConfirmPassword

the `isLoading` property in each of these have is used to control the button loader and is actually used in all of them

Reply:
> If by Authentication context you mean the prop in `pages/Authentication/index.tsx`, that is actually totally different. I dropped a comment to rename of the interface for that component's prop.

Maybe you can rename this particular interface from  `AuthenticationProps` to `AuthenticationProps`